pinot-core/src/test/java/org/apache/pinot/core/segment/index/readers/forward/FixedBitSVForwardIndexReaderV2Test.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.segment.index.readers.forward;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Random;
+import org.apache.commons.io.FileUtils;
+import org.apache.pinot.core.io.writer.impl.FixedBitSVForwardIndexWriter;
+import org.apache.pinot.core.segment.memory.PinotDataBuffer;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+
+public class FixedBitSVForwardIndexReaderV2Test {
+  private static final File INDEX_DIR = new File(FileUtils.getTempDirectory(), "FixedBitIntReaderTest");
+  private static final int NUM_VALUES = 99999;
+  private static final int NUM_DOC_IDS = 10000;
+  private static final Random RANDOM = new Random();
+
+  private final int[] _sequentialDocIds = new int[NUM_DOC_IDS];
+  private final int[] _sparseDocIds = new int[NUM_DOC_IDS];
+  private final int[] _lastSequentialDocIds = new int[NUM_DOC_IDS];
+  private final int[] _dictIdBuffer = new int[NUM_DOC_IDS];
+
+  @BeforeClass
+  public void setUp()
+      throws IOException {
+    FileUtils.forceMkdir(INDEX_DIR);
+
+    int sequentialDocId = RANDOM.nextInt(32);
```

Comment:
Would it be better to use explicit values instead of random?
For example, we know all the corner cases would be around bits being split across read boundaries. We can then programmatically generate all the splits.
We could have random as well as explict.

Reply:
That's why I explicitly tested reading the last doc ids (`_lastSequentialDocIds`).
Modified the test to test sequentialDocId starting from 0 - 31